index.bs

```diff
@@ -176,19 +185,101 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
 The syntax and semantics of the <dfn>AV1 Item Configuration Property</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
 
   - [=Sequence Header OBUs=] should not be present in the [=AV1CodecConfigurationBox=].
-  - If a [=Sequence Header OBU=] is present in the [=AV1CodecConfigurationBox=], it shall match the  [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
+  - If a [=Sequence Header OBU=] is present in the [=AV1CodecConfigurationBox=], it shall match the [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
   - The values of the fields in the [=AV1CodecConfigurationBox=] shall match those of the [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
   - [=Metadata OBUs=], if present, shall match the values given in other item properties, such as the PixelInformationProperty or ColourInformationBox.
 
 This property shall be marked as essential.
 
-<h4 id ="other-item-property">Other Item Properties</H4>
+<h4 id ="other-item-property">Other Item Properties</h4>
 
 In addition to the Image Properties defined in [[!HEIF]], such as 'colr', 'pixi' or 'pasp', [=AV1 image items=] MAY also be associated with 'clli', 'cclv' and 'mdcv' introduced in [[!MIAF]].
 
 In general, it is recommended to use properties instead of [=Metadata OBUs=] in the [=AV1 Item Configuration Property=].
 
-For multi-layered images, the LayerSelectorProperty may be present as defined in [[!HEIF]].
+<h3 id ="layered-items">AV1 Layered Image Items</h3>
+
+<h4 id ="layered-items-overview">Overview</h4>
+
+[[!AV1]] supports encoding a frame using multiple spatial layers. A spatial layer may improve the resolution or quality of the image decoded based on one or more of the previous layers. A layer may also provide an image that does not depend on the previous layers. Additionally, not all layers are expected to produce an image meant to be rendered. Some decoded images may be used only as intermediate decodes. Finally, layers are grouped into one or more [=Operating Points=]. The [=Sequence Header OBU=] defines the list of [=Operating Points=], provides required decoding capabilities, and indicates which layers form each [=Operating Point=].
+
+[[!AV1]] delegates the selection of which [=Operating Point=] to process to the application. AVIF defines the [=OperatingPointSelectorProperty=] to control this selection. In the absence of an [=OperatingPointSelectorProperty=] associated with an [=AV1 Image Item=], the AVIF renderer is free to process any [=Operating Point=] present in the [=AV1 Image Item Data=]. In particular, when the [=AV1 Image Item=] is composed of a unique [=Operating Point=], the [=OperatingPointSelectorProperty=] should not be present. If an [=OperatingPointSelectorProperty=] is associated with an [=AV1 Image Item=], the <code>op_index</code> field indicates which [=Operating Point=] is expected to be presented for this item.
```

Comment:
`field indicates which [=Operating Point=] is expected to be presented for this item.`
Do we want to make this a shall? If the file creator selected a specific operating point due to the other operating points containing completely different images, not decoding the specified operating point could get very weird.

Same goes the `lsel` wording.

Reply:
I refrain from 'shall' statements on player/reader behaviors and keep them for files (i.e. can be validated by a tool), but this can be changed.

Reply:
> I refrain from 'shall' statements on player/reader behaviors and keep them for files
You're right, that's probably the correct approach. It should be enough to explain what the expected player behaviour is. A HEIF box GUI viewer may want to show everything in the file, and we don't want to say that that is not allowed.